code/src/components/Header.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import moment from 'moment'
+import styled from 'styled-components'
+
+const Header = () => {
+  const date = moment().format('MMM Do'); 
```

Comment:
Though moment is very nice and useful, we might want to change this in the future as it's not maintained anymore :)